Cargo.toml

```diff
@@ -5,10 +5,13 @@ edition = "2018"
 license = "Apache-2.0"
 name = "zhang"
 version = "0.0.1-alpha.1"
+resolver = "2"
 
 build = "build.rs"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+frontend = ["rust-embed"]
```

Comment:
My thought is actually to use a flag to guard the whole server part (including HTTP backend). This will drastically decrease the crates needed to be downloaded and compiled when running tests (and in the development of core functionality).

Reply:
https://github.com/zhang-accounting/zhang/issues/126#issuecomment-1479027703

next step I gonna do is to separate the repo into mulitple lib, so dev can focus on one small part of it, do not need compile the while project while development.

Reply:
This would be surely even better